locale/en/blog/community/quality-with-speed.md

```diff
@@ -0,0 +1,316 @@
+there are specific concerns or for larger changes where the community
+believes more soak time is required.  One key exception is
+that sermver breaking changes will not be backported until the
+next major version (ex 5 -> 6).  This includes v8 and other
+components such that the expectation is that an application/module
+written to run on a major level will continue to do so.
+
+These releases are documented in the changelog so
+there is more visibility with respect to the changes in the release.
+Current releases are created roughly every 1-2 weeks.
+
+In addition to the the regular Node.js unit tests, CITGM(see
+later sections) is run on Current releases.
+
+If you want to try out the latest with a reasonable expectation
+that your application will continue to run, these are the releases
+to use.
+
+## LTS
+
+Once changes have been proven in the Current stream, they are candidates
+for the LTS stream.  Changes are limited to:
+
+* Bug fixes
+* Security updates
+* Non-semver-major npm updates
+* Relevant documentation updates
+* Certain performance improvements where the risk of
+  breaking existing applications is minimal
+* Changes that introduce large amount of code churn where
+  the risk of breaking existing applications is low and
+  where the change in question may significantly ease the
+  ability to backport future changes due to the reduction in diff noise.
+
+Further, in the later half of an LTS release (Maintenance), only
+**critical** bugs and **critical** security fixes will be included.
+
+Like Current releases, CITGM(see
+later sections) is run on Current releases.  In addition we also
+track performance through nightly benchmarks reported on
+[benchmarking.nodejs.org](benchmarking.nodejs.org) (See later sections).
+
+You can read more about the LTS releases [here](https://github.com/nodejs/lts).
+
+If you want the best level of stability/quality for your production
+applications these are the releases to use.
+
+# Change flow processes
+
+We've already touched on this in the discussion on the different release
+types but we'll expand on this strategy here.
+
+The main idea is that as changes flow from Nightlies, to Stable, to LTS we
+increase the following:
+
+* scrutiny
+* time
+
+Changes going into master are well reviewed and time is allowed
+(minimum 48 to 72 hours) for as many community members to comment and review.
+However, as we all know some problems will still get through.
+
+Before changes are pulled into Current from the Nightly, they will have
+spent more time in master where intermittent issues may surface in the
+ongoing regressions runs and providing time where users may more fully
+exercise the release and report issues.  Further, there is additional
+review/sanity check that they are not breaking as they are pulled over to
+Current.
+
+Similarly, before changes are pulled into an LTS update release,
+they must have been in a
+Current release version for at least a week, most often longer.  This provides
```

Comment:
most -> "and are often left"  

existing wording reads to me as "often must", which just leads me asking, how is it decided that it must be at least a week, and how is it decided it must be longer than a week? but I think what is mean it is it MUST be at least a week. It is OFTEN longer than a week if the change is considered risky